components/x-audio/src/redux/__tests__/player-logic.test.js

```diff
@@ -33,6 +33,24 @@ describe('actions and reducer', () => {
 		expect(updatedState).toMatchSnapshot();
 	});
 
+	test('Ended action sets ended to false', () => {
```

Comment:
@benbarnett Sorry for the late review... I know this PR is already merged. This text is incorrect?
'Ended action sets ended to false' => 'Ended action sets ended to **true**'

Reply:
Whoops! Yes you're right, thanks. Fixed in e436af48